test/freenet/client/filter/ContentFilterTest.java

```diff
@@ -3,28 +3,24 @@
  * http://www.gnu.org/ for further details of the GPL. */
 package freenet.client.filter;
 
+import junit.framework.TestCase;
```

Comment:
The tests don't pass without those changes... I've removed the dependency on l10n from the test